src/coverlet.collector/DataCollection/CoverletSettingsParser.cs

```diff
@@ -75,19 +76,20 @@ private string ParseTestModule(IEnumerable<string> testModules)
         }
 
         /// <summary>
-        /// Parse report format
+        /// Parse report formats
         /// </summary>
         /// <param name="configurationElement">Configuration element</param>
-        /// <returns>Report format</returns>
-        private string ParseReportFormat(XmlElement configurationElement)
+        /// <returns>Report formats</returns>
+        private string[] ParseReportFormats(XmlElement configurationElement)
         {
-            string format = string.Empty;
+            string[] formats = Array.Empty<string>();
             if (configurationElement != null)
             {
                 XmlElement reportFormatElement = configurationElement[CoverletConstants.ReportFormatElementName];
-                format = reportFormatElement?.InnerText?.Split(',').FirstOrDefault();
+                formats = reportFormatElement?.InnerText?.Split(',')
```

Comment:
use string split options to remove empty record so we can simply parse and check at the end https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=netcore-2.2#System_String_Split_System_String_System_StringSplitOptions_

Reply:
Done